Testscripts/Linux/check_triggered_nmi.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+########################################################################
+#
+# nmi_verify_interrupt.sh
+# Description:
+#	This script was created to automate the testing of a Linux
+#	Integration services. This script will verify if a NMI sent
+#	from Hyper-V is received  inside the Linux VM, by checking the
+#	/proc/interrupts file.
+#	The test performs the following steps:
+#	 1. Make sure we have a constants.sh file.
+#	 2. Looks for the NMI property of each CPU.
+#	 3. For 2012R2, verifies if each CPU has received a NMI.
+#	 4. For 2016, verifies only cpu0 has received a NMI.
+#
+################################################################
+
+set -e
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function main() {
+
+    UpdateSummary "Hyper-V build number: $BuildNumber"
```

Comment:
We started to use LogMsg instead of UpdateSummary in all the scripts. Not sure if Ju will be ok with UpdateSummary